crates/matrix-sdk-crypto-nodejs/src/requests.rs

```diff
@@ -56,7 +56,7 @@ pub struct KeysQueryRequest {
 
     /// A JSON-encoded object of form:
     ///
-    /// ```
+    /// ```json
```

Comment:
Right, hit that one as well but forgot to push the patch I have somewhere.

Reply:
Oops 🙊